source/platforms/GitLab.ts

```diff
@@ -178,6 +197,17 @@ class GitLab implements Platform {
     )
   }
 
+  getDangerThreads = async (dangerID: string) => {
+    const { id: dangerUserId } = await this.api.getUser()
+    const threads = await this.api.getMergeRequestThreads();
```

Comment:
This could be resolved with Promise.all

```suggestion
    const [{ id: dangerUserId }, threads] = await Promise.all([this.api.getUser(), this.api.getMergeRequestThreads()])
```